src/lib/bootloader/sysconfig.rb

```diff
@@ -120,14 +120,12 @@ def ensure_file_exists_in_target
       return if File.exist?(File.join(destdir, "/etc/sysconfig"))
 
       Yast::WFM.Execute(Yast::Path.new(".local.mkdir"),
-        File.join(destdir, "/etc/sysconfig")
-                       )
+        File.join(destdir, "/etc/sysconfig"))
       Yast::WFM.Execute(Yast::Path.new(".local.bash"),
-        "touch #{destdir}/etc/sysconfig/bootloader"
-                       )
+        "touch #{destdir}/etc/sysconfig/bootloader")
     end
 
-    def temporary_target_agent(&block)
```

Comment:
I think the speedup is negligible, I mean we are not even iterating over a container. Scrap this kind of speedups.